pages/browse-tasks.tsx

```diff
@@ -60,14 +60,16 @@ const extraColumns: GridColDef[] = [
     renderHeader: () => <TableHeader>Rating</TableHeader>,
     renderCell: (params: GridValueGetterParams) =>
       params.row.rating >= 0 ? (
-        <StyledRating
-          readOnly
-          value={params.row.rating}
-          size="large"
-          precision={0.5}
-          icon={<StarRoundedIcon fontSize="inherit" />}
-          emptyIcon={<StarOutlineRoundedIcon fontSize="inherit" />}
-        />
+        <Box ml="-7px">
+          <StyledRating
+            readOnly
+            value={params.row.rating}
+            size="large"
+            precision={0.5}
```

Comment:
I made an earlier comment about rounding the rating and see here that there's this precision variable. Wanted to clarify if this is doing the "rounding" visually here closer to rendering?

Reply:
Yes, this is doing the rounding to the nearest 0.5